report/src/test/kotlin/com/hartwig/actin/report/interpretation/MolecularDriverEntryFactoryTest.kt

```diff
@@ -108,11 +109,17 @@ class MolecularDriverEntryFactoryTest {
         )
         assertVariantType(oncoCav.copy(proteinEffect = ProteinEffect.LOSS_OF_FUNCTION), "Mutation (Cancer-associated variant)")
         assertVariantType(oncoCav.copy(proteinEffect = ProteinEffect.GAIN_OF_FUNCTION), "Mutation (Gain of function)")
-        assertVariantType(oncoNoCav.copy(proteinEffect = ProteinEffect.UNKNOWN), "Mutation (No known cancer-associated variant)")
-        assertVariantType(oncoNoCav.copy(proteinEffect = ProteinEffect.NO_EFFECT), "Mutation (No protein effect)")
-        assertVariantType(oncoNoCav.copy(proteinEffect = ProteinEffect.NO_EFFECT_PREDICTED), "Mutation (No protein effect)")
-        assertVariantType(oncoNoCav.copy(proteinEffect = ProteinEffect.LOSS_OF_FUNCTION), "Mutation (No known cancer-associated variant)")
-        assertVariantType(oncoNoCav.copy(proteinEffect = ProteinEffect.GAIN_OF_FUNCTION), "Mutation (Gain of function)")
+        assertVariantType(
+            oncoNoCav.copy(proteinEffect = ProteinEffect.UNKNOWN),
+            "Mutation (No known cancer-associated variant, high driver)"
+        )
+        assertVariantType(oncoNoCav.copy(proteinEffect = ProteinEffect.NO_EFFECT), "Mutation (No protein effect, high driver)")
+        assertVariantType(oncoNoCav.copy(proteinEffect = ProteinEffect.NO_EFFECT_PREDICTED), "Mutation (No protein effect, high driver)")
+        assertVariantType(
+            oncoNoCav.copy(proteinEffect = ProteinEffect.LOSS_OF_FUNCTION),
+            "Mutation (No known cancer-associated variant, high driver)"
```

Comment:
This test looks a bit weird to me, it is essentially a bug in our datamodel. How can a variant be non-CAV yet have loss-of-function? I think for now I'd just remove these inconsistent tests and at same point improve the datamodel